scout/server/utils.py

```diff
@@ -40,6 +40,11 @@ def public_endpoint(function):
 
 def institute_and_case(store, institute_id, case_name=None):
     """Fetch insitiute and case objects."""
+
+    # Check if user is authenticated. This should prevent the access to un-authenticated users and registered users with expired session
+    if current_user.is_authenticated is False:
```

Comment:
But.. why would you need this? I can perhaps understand wanting to issue a 401 specifically, but the auth should be handled already for all endpoints under Flask-Login? Specifically by the server app before_request decorator that you edited? Please give a counterexample otherwise if I'm missing something! 

I'm a little worried constructs like this would cause later confusion about where auth is actually handled!

Reply:
No, I agree, that's why I wrote that it's probably an overkill. Shall we remove it?

Reply:
Aye, then we agree, and I see the reason. I missed the overkill caveat - better safe then sorry is not a bad principle. Personally, I think it better to keep it clean and correct. This is an area where worry and confusion often kicks in.